lib/hubstats/events_handler.rb

```diff
@@ -22,18 +22,20 @@ def route(payload, type)
       end
     end
 
-    # Public - Gets the information for the new PR, makes the new PR, grabs the labels, and makes new labels
+    # Public - Gets the information for the PR, creates/updates the new PR, grabs the labels, and makes new labels
     #
     # payload - the information that we will use to get data off of
     #
-    # Returns - nothing, but makes the new PR and adds appropriate labels
+    # Returns - nothing, but creates/updates the PR and adds appropriate labels
     def pull_processor(payload)
       pull_request = payload[:pull_request]
       pull_request[:repository] = payload[:repository]
       new_pull = Hubstats::PullRequest.create_or_update(pull_request.with_indifferent_access)
       repo_name = Hubstats::Repo.where(id: new_pull.repo_id).first.full_name
-      labels = Hubstats::GithubAPI.get_labels_for_pull(repo_name, new_pull.number )
+      labels = Hubstats::GithubAPI.get_labels_for_pull(repo_name, new_pull.number)
+      process_label_change(labels, payload) if payload[:action].include?('labeled') # When a new label is added/removed
```

Comment:
@anfleene I was looking into this and noticed that `add_labels` is also called by [GithubAPI](https://github.com/sportngin/hubstats/blob/master/lib/hubstats/github_api.rb#L302) and it accepts an array of labels. So I am going to leave that method as-is and add an `update_labels` method to just handle the addition/deletion of a label.

Sound good?